src/partials/footer.html

```diff
@@ -1,29 +1,30 @@
 <footer>
-    <div>
-    <h2>HOMEMADE ICE CREAM MADE<span> WITH PASSION</span></h2>
-      <ul>
-        <li>
-          <a href="https:/github.com/" target="_blank"
-            ><svg><use></use></svg></a
-          >
-        </li>
-        <li>
-          <a href="https://telegram.org/" target="_blank"
-            ><svg><use>tel</use></svg></a
-          >
-        </li>
-        <li>
-          <a href="https://facebook.com/" target="_blank"
-            ><svg><use>fb</use></svg></a
-          >
-        </li>
-        <li>
-          <a  href="tel:+610383766284"
-            >Feel free to contact <span>+61 (0) 383 766 284</span></a
-          >
-        </li>
-      </ul>
-    </div>
-    <span>&copy; 2019 - IceCream. All Rights Reserved.</span>
+  <div class="footer">
```

Comment:
to jest div dla container, tylko taką klasę możesz tu umieścić, klasę footer daj do tagu footer wyżej

Reply:
dlaczego tylko taką?  "Używanie klasy container nie jest obowiązkiem, jest jedynie dobrą praktyką stosowaną przez wielu developerów. "